src/types/widget.ts

```diff
@@ -7,10 +8,12 @@ import {
 
 export interface InitOptions {
   instantSearchInstance: InstantSearch;
+  parent: Index | null;
   state: SearchParameters;
   helper: Helper;
   templatesConfig: object;
   createURL(state: SearchParameters): string;
+  onHistoryChange(callback: (state: Partial<SearchParameters>) => void): void;
```

Comment:
Why did we add `onHistoryChange` back?

Reply:
https://github.com/algolia/instantsearch.js/pull/3892#discussion_r297723400